contracts/mocks/WeedTokenMock.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import "../token/SideToken.sol";
+
+contract WeedTokenMock is SideToken, UUPSUpgradeable {
```

Comment:
can WeedTokenMock be derived from WeedToken

Reply:
Not without adding more unnecessary code to the WeedToken